src/components/ChatLog.js

```diff
@@ -0,0 +1,33 @@
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+
+    const chatComponents = props.entries.map(entry => {
+        return (
+            <div key={entry.id}>
+                <ChatEntry 
+                id={entry.id} 
+                sender={entry.sender} 
+                body={entry.body} 
+                timeStamp={entry.timeStamp} 
+                liked={entry.liked} 
+                toggleLike={props.toggleLike}>
+                </ChatEntry>
+            </div>
+        );
+    });
+
+    return (
+        <section>
+            <ul>
+                {chatComponents}
+            </ul>
+        </section>
+    );
+};
+
+
+
```

Comment:
Remember to put your props-types checking for this one. You could use the `arrayOf` and then `shape`  functionality to be very explicit about the data structure you are expecting to pass through here.